CMakeLists.txt

```diff
@@ -127,6 +127,13 @@ else()
     message(STATUS "UFE not found.")
 endif()
 
+find_package(MAYAUSD OPTIONAL_COMPONENTS)
```

Comment:
Nit-pick: why the all-caps naming?  UFE and USD packages are uppercase, because their name are uppercase, but MayaUsd is not --- e.g. look at the Python and Qt6 package names